app/components/primer/underline_nav_component.rb

```diff
@@ -5,27 +5,44 @@ module Primer
   # underlined selected state, typically used for navigation placed at the top
   # of the page.
   class UnderlineNavComponent < Primer::Component
+    include ViewComponent::SlotableV2
+
     ALIGN_DEFAULT = :left
     ALIGN_OPTIONS = [ALIGN_DEFAULT, :right].freeze
 
-    with_content_areas :body, :actions
+    #  Use the body for the navigation items
+    #
+    # @param system_arguments [Hash] <%= link_to_system_arguments_docs %>
+    renders_one :body, lambda { |**system_arguments|
+      system_arguments[:classes] = class_names("UnderlineNav-body list-style-none", system_arguments[:classes])
+
+      Primer::BaseComponent.new(tag: :ul, **system_arguments) { content }
+    }
+
+    #  Use actions for a call to action
+    #
+    # @param system_arguments [Hash] <%= link_to_system_arguments_docs %>
+    renders_one :actions, lambda { |**system_arguments|
+      system_arguments[:tag] ||= :span
+      Primer::BaseComponent.new(**system_arguments) { content }
```

Comment:
This adds a span that we did not have before. Is that okay? Is there a better way to handle this?

Reply:
looking at https://primer.style/css/components/navigation#underline-nav I feel that we are missing a `<div class="UnderlineNav-actions">`, which could be this slot